substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -258,7 +258,7 @@ impl<T: Config> Pallet<T> {
 		// This is how much validator + nominators are entitled to.
 		let validator_total_payout = validator_total_reward_part * era_payout;
 
-		let validator_commission = EraInfo::<T>::get_validator_commission(era, &ledger.stash);
+		let validator_commission = EraInfo::<T>::get_validator_commission(era, &stash);
 		// total commission validator takes across all nominator pages
 		let validator_total_commission_payout = validator_commission * validator_total_payout;
```

Comment:
is this math safe?

Reply:
Fixed here https://github.com/paritytech/polkadot-sdk/pull/2435